worlds/inscryption/__init__.py

```diff
@@ -0,0 +1,122 @@
+from .Options import InscryptionOptions
+from .Items import act1_items, act2_items, act3_items, filler_items, base_id, InscryptionItem, ItemDict
+from .Locations import act1_locations, act2_locations, act3_locations, regions_to_locations
+from .Regions import inscryption_regions_all, inscryption_regions_act_1
+from typing import Dict, Any
+from . import Rules
```

Comment:
```suggestion
import .Rules
```
Not 100% sure this is correct since I'm not in front of an IDE atm

Reply:
I'm assuming this should be okay as it is? Gonna mark this as resolved if that's the case.